src/components/SiderMenu/SiderMenu.tsx

```diff
@@ -0,0 +1,73 @@
+import React, { useState, Suspense } from 'react'
+import { Layout } from 'antd'
+import classNames from 'classnames'
+import Link from 'umi/link'
+import styles from './index.less'
```

Comment:
to avoid typescript complains about this line (if we want to compile and typecheck this all sources without using babel -- and it's good to have the possibility to not relay on babel) we need to use require for all non typescript code. So here we need to use `require`:

```
const styles = require(...)
```

Reply:
- [ ] use require here

We can think how to solve it in other way later, I was already thinking about it last week (when adding support for babel), and created this task: https://trello.com/c/M21cmwQP/18-find-a-way-to-userimport-syntax-for-less-files-and-svg-files-in-tsx

Reply:
read the trello task